geth/peers/peerpool.go

```diff
@@ -149,8 +145,10 @@ func (p *PeerPool) handleServerPeers(server *p2p.Server, events <-chan *p2p.Peer
 				if p.stopOnMax && p.handleDroppedPeer(server, event.Peer) {
 					retryDiscv5 = time.After(0)
 				}
+				p.signalHandler.DiscoverySummary(server.PeersInfo())
```

Comment:
Is it possible that `PeerInfo` has also inbound connections?

Reply:
not on mobile. actually, we need to disable listener on mobile if we want to prevent such connections from happening, but that's another topic.

but even if connection is inbound it is still valid from app perspective

Reply:
IMHO there's no separation between peers in server.